### PR TITLE
Fix nightly build instruction on mxnet webpage

### PR DESCRIPTION
**Pull request by apeforest**
## Description ##
As title

@leezu @aaronmarkham Please review.

### CODE REVIEW

File: docs/static_site/src/_includes/get_started/windows/python/gpu/pip.md [hunk 1]
```diff
@@ -64,7 +64,7 @@ $ pip install mxnet-cu80==0.11.0
 <div class="master">
 
 {% highlight bash %}
-$ pip install mxnet-cu101 --pre
+$ pip install --pre mxnet-cu102 -f https://dist.mxnet.io/python/all
```
Comment (sandeep-krishnamurthy) [inline docs/static_site/src/_includes/get_started/windows/python/gpu/pip.md]: is cu10.2 upgrade here intended?

Comment (apeforest) [reply docs/static_site/src/_includes/get_started/windows/python/gpu/pip.md]: I think in nightly build we already supported the latest cuda. @leezu please confirm.

Comment (leezu) [reply docs/static_site/src/_includes/get_started/windows/python/gpu/pip.md]: I think it's reasonable to update to cu10.2. Cu10.2 fixes cuda bugs in cu10.1, so we should recommend 10.2

**Comment by leezu** [comment]
Retriggered Windows GPU

**Pull request merged**